src/cclib/writer/ccwrite.py

```diff
@@ -67,7 +68,10 @@ def ccwrite(ccobj, outputtype=None, outputdest=None, returnstr=False,
         # Avoid passing multiple times into the main call.
         del kwargs['jobfilename']
 
-    outputobj = outputclass(ccdata, jobfilename=jobfilename, *args, **kwargs)
+    if cjsonterse:
```

Comment:
Since you are passing kwargs, just add the terse argument to it to avoid two of the same calls. Or change `cjsonterse` into `terse` as noted above and always pass it.